variables.tf

```diff
@@ -70,6 +70,11 @@ variable "allow_major_version_upgrade" {
   default     = "false"
 }
 
+variable "force_ssl" {
+  description = "Enforce SSL connections, set to 1 to enable"
```

Comment:
> set to 1 to enable
For this module it's a boolean, can you please update?

Reply:
Fixed.